launch/driver.launch

```diff
@@ -13,10 +13,10 @@ Licensed under the MIT License.
   <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
 
   <arg name="depth_enabled"           default="true" />           <!-- Enable or disable the depth camera -->
-  <arg name="depth_mode"              default="WFOV_UNBINNED" />  <!-- Set the depth camera mode, which affects FOV, depth range, and camera resolution. See Azure Kinect documentation for full details. Valid options: NFOV_UNBINNED, NFOV_2X2BINNED, WFOV_UNBINNED, WFOV_2X2BINNED -->
+  <arg name="depth_mode"              default="WFOV_UNBINNED" />  <!-- Set the depth camera mode, which affects FOV, depth range, and camera resolution. See Azure Kinect documentation for full details. Valid options: NFOV_UNBINNED, NFOV_2X2BINNED, WFOV_UNBINNED, WFOV_2X2BINNED, and PASSIVE_IR -->
   <arg name="color_enabled"           default="true" />           <!-- Enable or disable the color camera -->
   <arg name="color_format"            default="bgra" />           <!-- The format of RGB camera. Valid options: bgra, jpeg -->
-  <arg name="color_resolution"        default="1536P" />          <!-- Resolution at which to run the color camera. Valid options: 720P, 1080P, 1440P, 1536P, 2160P, 3072P -->
+  <arg name="color_resolution"        default="720P" />          <!-- Resolution at which to run the color camera. Valid options: 720P, 1080P, 1440P, 1536P, 2160P, 3072P -->
```

Comment:
Good call

Reply:
1536P isn't a bad choice either when the use case is to create rgb pointclouds because this resolution has a greater overlap with the depth camera field of view.
The 4:3 resolutions cover the whole NFOV depth image:
https://docs.microsoft.com/en-us/azure/kinect-dk/hardware-specification#camera-field-of-view

Reply:
That's a great observation, thank you for that insight. 
I've been fielding questions about performance concerns with the default configuration, which is where my comment came from. @helenol - Your call.

Reply:
@ooeygui 
The bottleneck at the moment is transfering the SDK pointcloud data to the ROS message as discussed here:
https://github.com/microsoft/Azure_Kinect_ROS_Driver/issues/56

Obviously its gets worse with greater resoultions, but at least its good to know that the transformation from depth to color and point cloud data generation from the sdk itself isn't the bottleneck.

Reply:
Thanks for the discussion! I'll revert back to 1536P, makes sense to me with the wider vertical FoV. Thanks for bringing #56 to my attention, I'll take a quick look tomorrow I think we can make some improvements in that department. :)